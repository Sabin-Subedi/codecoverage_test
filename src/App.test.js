import { render, screen, cleanup } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import App from "./App";

afterEach(cleanup);

test("Test theme button toggle", () => {
  const { container } = render(<App />);
  const button = screen.getByText(/Current theme/i);

  userEvent.click(button);
  expect(button).toHaveTextContent(/dark/i);

  expect(container).toMatchSnapshot();
});
