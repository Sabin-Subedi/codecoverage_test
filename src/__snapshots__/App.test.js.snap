// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test theme button toggle 1`] = `
<div>
  <button>
    Current theme: 
    dark
  </button>
</div>
`;
